File: go/raindrops/raindrops.go
package raindrops

import (
	"fmt"
	"strings"
)

const testVersion = 2

func Convert(num int) string {

	var s strings.Builder

	if num%3 == 0 {
		s.WriteString("Pling")
	}

	if num%5 == 0 {
		s.WriteString("Plang")
	}

	if num%7 == 0 {
		s.WriteString("Plong")
	}

	if s.Len() == 0 {
		return fmt.Sprintf("%d", num)
	}

	return s.String()
}

File: python/raindrops/raindrops.py
def convert(number):
    computed_value = ''

    if number%3 == 0:
        computed_value += "Pling"

    if number%5 == 0:
        computed_value += "Plang"

    if number%7 == 0:
        computed_value += "Plong"

    if len(computed_value) == 0:
        return str(number)

    return computed_value
